.circleci/config.yml

```diff
@@ -61,29 +61,29 @@ jobs:
             - browser-tools/install-browser-tools
 
             - restore_cache:
-                key: deps1-{{ .Branch }}-{{ checksum "tests/requirements.txt" }}-{{ checksum "demos/requirements.txt" }}-{{ checksum "requirements.txt" }}
+                key: deps2-{{ .Branch }}-{{ checksum "tests/requirements.txt" }}-{{ checksum "demos/requirements.txt" }}-{{ checksum "requirements.txt" }}
 
             - run:
                 name: Create virtualenv
                 command: |
-                    python -m venv venv
+                    python -m venv dash-deck-venv
```

Comment:
Is that true, these `cimg/python` images have a default `venv` that conflicts with the one we're making? Not in some parent directory? That's extremely unfortunate if true, and will have an impact all across Dash.

Reply:
it is not true, I was tricked by the error message